src/Bicep.LangServer/Handlers/BicepDefinitionHandler.cs

```diff
@@ -49,28 +49,28 @@ public override Task<LocationOrLocationLinks> Handle(DefinitionParams request, C
             {
                 return Task.FromResult(new LocationOrLocationLinks());
             }
+
             var result = this.symbolResolver.ResolveSymbol(request.TextDocument.Uri, request.Position);
+
             // No parent Symbol: ad hoc syntax matching
-            if (result is null)
-            {
-                return HandleUnboundSymbolLocationAsync(request, context);
-            }
-            // Declared symbols: go to definition
-            else if (result.Symbol is DeclaredSymbol declaration)
-            {
-                return HandleDeclaredDefinitionLocationAsync(request, result, declaration);
-            }
-            // Object property: currently only used for module param goto
-            else if (result.Origin is ObjectPropertySyntax objectPropertySyntax)
+            switch (result)
```

Comment:
[nit]: Can further simplify this with pattern matching.

Reply:
Which part?

Reply:
I meant the switch statement can be changed to a switch expression:
```
return result switch
{
    null => HandleUnboundSymbolLocationAsync(request, context),
    ...
}
```

Reply:
Oh good pt. Just pushed an update with the change.